.github/copilot-instructions.md

```diff
@@ -0,0 +1,347 @@
+The documentation should include explicit naming patterns for gene-related diseases:
+
+```markdown
+## Gene-Disease Naming Conventions
+
+For monogenic diseases, MONDO follows the pattern: `{GENE}-related {disease description}`
+
+Examples:
+- ALG8-related autosomal dominant polycystic kidney and/or liver disease
+- MYCBP2-related developmental delay with corpus callosum defects
+- TUBB4B-related ciliopathy
+
+This pattern should be used even when users request alternative formats like "disease - GENE" format.
+```
+
+## 2. Synonym Attribution and Citation Requirements
+
+The current documentation lacks specific guidance on synonym citations:
+
+```markdown
+## Synonym Citation Requirements
+
+ALL synonyms must include proper citations:
+
+**Correct examples:**
+```
+synonym: "EBV-associated lymphoproliferative disorder" EXACT [PMID:38882456]
+synonym: "MDCD" EXACT ABBREVIATION [PMID:36200388]
+synonym: "CHD4-related neurodevelopmental disorder" EXACT [https://orcid.org/0000-0001-9310-0163, Orphanet:653712]
+```
+
+**Never use empty brackets:**
+```
+synonym: "term name" EXACT []  # INCORRECT
+```
+
+### ClinGen Label Handling
+When ClinGen provides preferred labels, use the CLINGEN_LABEL qualifier:
+```
+synonym: "Hajdu-Cheney syndrome-NOTCH2" EXACT CLINGEN_LABEL [https://clinicalgenome.org/affiliation/40066/]
+```
+```
+
+### Gene-Based Disease Pattern Requirements
+
+When following the disease_series_by_gene pattern:
+
+**Required logical structure:**
+```
+is_a: MONDO:0700092 ! neurodevelopmental disorder
+intersection_of: MONDO:0700092 ! neurodevelopmental disorder
+intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/23386 ! MYCBP2
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/23386 {source="PMID:36200388"} ! MYCBP2
+```
+
+**Definition template:**
+
+```
+def: "Any [parent disease class] in which the cause of the disease is a mutation in the [GENE] gene." [PMID:references]
+```
+```
+
+## 4. Parent Term Selection Guidelines
+
+### For Gene-Related Diseases
+- Use the most general appropriate disease category that encompasses all known phenotypes
+- For multi-system diseases, consider dual parentage rather than single broad syndromic parent
+- Example: For neuro-urological conditions, use both `nervous system disorder` AND `urinary system disorder`
+
+## 5. Source Attribution in Relationships
+
+## Source Attribution Requirements
+
+All logical axioms must include source attribution when based on literature:
+
+```
+is_a: MONDO:0700092 {source="PMID:36200388", source="https://orcid.org/0000-0001-9310-0163"} ! neurodevelopmental disorder
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/23386 {source="PMID:36200388"} ! MYCBP2
+```
+
+Multiple sources can be cited for the same relationship.
+
+## 6. Gene Identifier Verification
+
+## Gene Identifiers
+
+**CRITICAL**: Always verify gene identifiers using the official HGNC database:
```

Comment:
this should be updated to account for non-human animal diseases with a gene association since these gene identifiers are pulled from NCBI Gene. 

@katiermullen or @twhetzel to update